services/core-api/app/api/now_applications/resources/now_application_progress_resource.py

```diff
@@ -75,7 +94,19 @@ def put(self, application_guid, application_progress_status_code):
         if not existing_now_progress:
             raise NotFound('This progress object has not been created yet')
 
-        existing_now_progress.end_date = datetime.now(tz=timezone.utc)
+        start_date = data.get("start_date", None)
+        end_date = data.get("end_date", None)
+        date_override = data.get("date_override", False)
+        my_time = datetime.min.time()
+        if can_edit_now_dates() and date_override:
+            if start_date is not None:
```

Comment:
Same as prior validation comment

Reply:
added! but I added the validations to the end_date, 'end_date < start_date'